src/main/java/com/linkedin/kmf/services/ProduceService.java

```diff
@@ -77,14 +79,17 @@
   private final int _threadsNum;
   private final String _zkConnect;
   private final boolean _treatZeroThroughputAsUnavailable;
+  private final int _latencyPercentileMaxMs;
+  private final int _latencyPercentileGranularityMs;
 
   public ProduceService(Map<String, Object> props, String name) throws Exception {
     _name = name;
     ProduceServiceConfig config = new ProduceServiceConfig(props);
     _zkConnect = config.getString(ProduceServiceConfig.ZOOKEEPER_CONNECT_CONFIG);
     _brokerList = config.getString(ProduceServiceConfig.BOOTSTRAP_SERVERS_CONFIG);
     String producerClass = config.getString(ProduceServiceConfig.PRODUCER_CLASS_CONFIG);
-
+    _latencyPercentileMaxMs = config.getInt(ProduceServiceConfig.LATENCY_PERCENTILE_MAX_MS_CONFIG);
+    _latencyPercentileGranularityMs = config.getInt(ProduceServiceConfig.LATENCY_PERCENTILE_GRANULARITY_MS_CONFIG);
```

Comment:
Same type of comment here.  Histogram bin size or bucket width or interval size seems more clear to me.